### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by sandipgiri576** [jla-gardner/graph-pes#107]
If I want to change the loss function, then  I need to change the loss.py (custom loss function) file or is there any interactive way to use the function during the training process?
